test/drivers/test_cqrrp.cc

```diff
@@ -100,10 +100,9 @@ class TestCQRRP : public ::testing::Test
         printf("MAX COL NORM METRIC:    %14e\n", max_col_norm / col_norm_A);
         printf("FRO NORM OF (Q'Q - I):  %14e\n\n", norm_0 / std::sqrt((T) n));
 
-        T atol = std::pow(std::numeric_limits<T>::epsilon(), 0.75);
         ASSERT_NEAR(norm_AQR / norm_A,         0.0, atol);
         ASSERT_NEAR(max_col_norm / col_norm_A, 0.0, atol);
-        ASSERT_NEAR(norm_0, 0.0, atol);
+        ASSERT_NEAR(norm_0 / std::sqrt((T) n), 0.0, atol);
```

Comment:
Instead of ``ASSERT_NEAR( var1 / var2, 0.0, atol)``, please use ``ASSERT_LE(var1, atol*var2)``

Reply:
Done.